sound/soc/sof/intel/hda-loader.c

```diff
@@ -372,3 +372,148 @@ int hda_dsp_cl_boot_firmware(struct snd_sof_dev *sdev)
 	dev_err(sdev->dev, "error: load fw failed err: %d\n", ret);
 	return ret;
 }
+
+// skl/kbl enable core and code loader DMA has some difference with apl/cnl
+// add the APIs for skl/kbl
+static int cl_stream_prepare_skl(struct snd_sof_dev *sdev, unsigned int format,
```

Comment:
Is this really needed to set tag to 0?

Reply:
No, this is for debug, this function will be implement in next patch. and if not set a value, checkpatch will give a warning.